schema/workflow.jsonschema

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
I think keeping this in JSON is consistent with a concern that @maddenp-noaa brought up last week. 👍

Reply:
Not for this PR, but we might like to adopt a tool like `jq` to format any JSON documents we're maintaining, for consistency and to avoid any formatting nitpicking that might otherwise arise. (Running JSON through `jq` is also a good sanity check for basic correctness.)

Reply:
Excellent!